doc/integrator/multi_organization.rst.tmpl

```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
 - org2-registered
 - org2-intranet
 
+Internationalization
+--------------------
+
+We should have one localization file for each organization.
+
+The general workflow is:
+- The integrator will run `make update-client-po`.
+- This will run one `update-po` script for each organization with different environment variables.
+- We will get one po file set for each organization with the defined suffix.
+- The integrator needs to complete the po files with translations.
+- When we build the config Docker image it will convert all the po file in the corresponding JSON files
+  with the same suffix.
+- At runtime the `dynamic` view will return the appropriated URLs for the translations.
+
+Configuration
+.............
+
+Add this rule in the `geoportal/Makefile`:
+
+.. code::
+
+   .PHONY: update-client-po
+   update-client-po:
+       # Generate po file for org1
+       cd .. && SUFFIX=-org1 INTERFACES=desktop-org1,mobile-org1 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+       # Generate po file for org2
+       cd .. && SUFFIX=-org2 INTERFACES=desktop-org2,mobile-org2 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+
+       # Example with all environment variables
+       cd .. && \
+           SUFFIX=-test \
+           INTERFACES=desktop-test,mobile-test \
+           THEME_REGEX=.*-test \
+           GROUP_REGEX=.*-test \
+           WMSLAYER_REGEX=.*-test \
+           WMTSLAYER_REGEX=.*-test \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+
+All available environment variable that can be added in the `geoportal/Makefile` file to run the
```

Comment:
The environment variables that are used by the update-po command are the following: